app/controllers/IncomeController.scala

```diff
@@ -214,7 +214,8 @@ class IncomeController @Inject()(
 
                     case _ => throw new RuntimeException(s"Not able to found employment with id $empId")
                   }
-                case _ => throw new RuntimeException("Exception while reading employment and tax code details")
+                case _ =>
+                  errorPagesHandler.internalServerError("Exception while reading employment and tax code details")
```

Comment:
should we log any of the errors that may have occurred?

Reply:
I'm confused, so as a user where will I be directed?

Reply:
they will just go to the standard internal server error page @PerrettJ4

Reply:
@sergi-hmrc would we do this in a recover block or like how you have done yours?

Reply:
my bad this method logs... sorry